xinterpreters/xinterpreters/host/host_interpreter.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2022 XMOS LIMITED. This Software is subject to the terms of the
+# XMOS Public License: Version 1
+import sys
+import ctypes
+import numpy as np
+from enum import Enum
+from pathlib import Path
+
+from xinterpreters.base.base_interpreter import xcore_tflm_base_interpreter
+
+__PARENT_DIR = Path(__file__).parent.absolute()
+if sys.platform.startswith("linux"):
+    lib_path = str(__PARENT_DIR / "libs" / "linux" / "xtflm_python.so")
+elif sys.platform == "darwin":
+    lib_path = str(__PARENT_DIR / "libs" / "macos" / "xtflm_python.dylib")
+else:
+    raise RuntimeError("libxcore_interpreters is not supported on Windows!")
```

Comment:
We are missing Windows build changes in this file

Reply:
Added.